internal_docs/engine_architecture.md

```diff
@@ -1,10 +1,15 @@
 # Engine Architecture
 
-![Alt text](assets/tee_diagram.drawio.svg)
+Engine itself is responsible for forwarding tracks to proper endpoints.
+It is protocol agnostic i.e. it doesn't store information specific for 
+any of multimedia protocols or standards.
```

Comment:
As you can only forward RTP through RTC Engine is this statement still true?

Reply:
I think so, internal engine elements like tees don't store any WebRTC specific information

Reply:
According to our conversation, you can try to rephrase it or leave it as it is.